content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -1,17 +1,29 @@
 # Use models
 
 Whether you need to download your models to use them, or you're looking to set
-up some automation in CI/CD to deploy them, Iterative Studio provides these
+up some automation in CI/CD to deploy them, DVC Studio provides these
 capabilities.
 
 ## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using Studio [REST API], `dvc get`, or DVC [Python API].
+versions using the DVC Studio [REST API], DVC [Python API], use the
+`dvc artifacts get` command or directly download your model from DVC Studio.
```

Comment:
```suggestion
versions with DVC Studio from the web interface or in your code.
```

Reply:
I don't think we need to detail all the methods in the intro. Let's assume users come here wanting to use Studio to help them download. We can point below to DVC docs below for how to download with DVC,